config/sso.go

```diff
@@ -18,10 +18,14 @@ type SSOConfig struct {
 	Scopes        []string        `json:"scopes,omitempty"`
 	SessionExpiry metav1.Duration `json:"sessionExpiry,omitempty"`
 	// customGroupClaimName will override the groups claim name
-	CustomGroupClaimName string   `json:"customGroupClaimName,omitempty"`
-	UserInfoPath         string   `json:"userInfoPath,omitempty"`
-	InsecureSkipVerify   bool     `json:"insecureSkipVerify,omitempty"`
-	FilterGroupsRegex    []string `json:"filterGroupsRegex,omitempty"`
+	CustomGroupClaimName string `json:"customGroupClaimName,omitempty"`
+	UserInfoPath         string `json:"userInfoPath,omitempty"`
+	InsecureSkipVerify   bool   `json:"insecureSkipVerify,omitempty"`
+  FilterGroupsRegex    []string `json:"filterGroupsRegex,omitempty"`
```

Comment:
these are [failing lint](https://github.com/argoproj/argo-workflows/actions/runs/6165751418/job/16734776648?pr=11777) (`gofmt` needs to be run)